core/io/resource_format_binary.cpp

```diff
@@ -854,13 +854,7 @@ void ResourceInteractiveLoaderBinary::open(FileAccess *p_f) {
 	}
 
 	bool big_endian = f->get_32();
-#ifdef BIG_ENDIAN_ENABLED
-	endian_swap = !big_endian;
-#else
-	bool endian_swap = big_endian;
```

Comment:
That can change the value of `print_bl("endian swap: " + itos(endian_swap));` below. It may have been a bug in the first place to declare this as local, shadowing the member var, but I'd be cautious at any rate.

Reply:
You're right, I didn't remixed l realize that endian_swap was a member, I thought this was just broken. I'll look at this file again.

Reply:
hmm, print_bl() actually compiles to nothing. So this was all a bit pointless:

```c++
//#define print_bl(m_what) print_line(m_what)
#define print_bl(m_what)
```

I think maybe I need to put all of that behind a debug flag and get rid of print_bl.

Reply:
Well it seems to be a trick to enable some debug output when hacking that code.

Reply:
Yeah, but maybe we have accumulated too many different tricks like this over the years. :)